templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
```

Comment:
The link for 'Manufacturers' is a placeholder (`href="#"`). Consider updating it with the actual URL or removing it if not needed.